Fix broken link

teoli2003 opened the pull request:
Fixes #13788

teoli2003 committed:
Fix broken link

Fixes #13788

wbamberg reviewed (commented):
This page (https://developer.mozilla.org/en-US/docs/Web/Performance) is weird. First, that broken link appears twice, once under ["Beginner's tutorials"](https://developer.mozilla.org/en-US/docs/Web/Performance#beginners_tutorials) and again under ["Documents yet to be written"](https://developer.mozilla.org/en-US/docs/Web/Performance#documents_yet_to_be_written).

We should obviously delete it from the second section. That section contains many other broken links. We should either delete the whole section (my preference) or not make those `dt` elements contain links.

I think we should also move https://developer.mozilla.org/en-US/docs/Learn/Performance/javascript_performance to https://developer.mozilla.org/en-US/docs/Learn/Performance/JavaScript , which is a better URL for it and is consistent with pages like https://developer.mozilla.org/en-US/docs/Learn/Performance/CSS. Note that the [Learn module itself](https://developer.mozilla.org/en-US/docs/Learn/Performance#guides) also thinks the URL is https://developer.mozilla.org/en-US/docs/Learn/Performance/JavaScript.

Overall the content at https://developer.mozilla.org/en-US/docs/Web/Performance/ looks half-written :(. We have https://github.com/openwebdocs/project/issues/62 which might be a vehicle for improving things...

teoli2003 committed:
Move javascript_performance -> Performance/JavaScript

teoli2003 committed:
Remove javascript_performance from non-written list of documents

teoli2003 committed:
Fix Link to Performance/JavaScript in "In this module" sections of Learn/Performance

teoli2003 commented:
I agree:
- I renamed the page.
- Deleted the entry in "to be written".
- Updated the link to this page (all in the "In this module" section of the `Learn/Performance`pages.

Improving the Web/Performance page is needed but out of scope, I think.

teoli2003 committed:
Delete the To Be Written section
